### PR TITLE
Include closest legislator district office phone number

### LINKED ISSUE #66: Add "Who to Call" data to `initiate-action` endpoint
Screenshot from [most recent design](https://www.figma.com/file/PEEkfIkziGfLogDHxSex9K/CC-%2F%2F-Action-Tool?node-id=205%3A2):
image

Looks like the front-end will need the following info for each representative:
- Who to call
- DC Office number
- Local Office number

Would make sense to add this onto `initiate-action` endpoint. @kevincianfarini thoughts? Not sure where this data lives, or if there can be more than 2 numbers to call for each representative.

### PR DESCRIPTION
**Pull request by kevincianfarini**
Closes #66

### CODE REVIEW

File: backend/src/main/kotlin/com/climatechangemakers/act/feature/findlegislator/model/Legislator.kt [hunk 1]
```diff
@@ -7,7 +7,7 @@ import kotlinx.serialization.Serializable
   val name: String,
   val role: LegislatorRole,
   val siteUrl: String,
-  val phone: String,
+  val phoneNumbers: List<String>,
```
Comment (kevincianfarini) [inline backend/src/main/kotlin/com/climatechangemakers/act/feature/findlegislator/model/Legislator.kt:10]: @jacknugent this is API breaking :/. Sorry about that.

File: frontend/src/models/ActionInfo.ts [hunk 2]
```diff
@@ -5,6 +5,7 @@ export type ActionInfo = {
         area: {
             state: string;
             districtNumber?: number;
+            districtPhoneNumber?: string;
```
Comment (kevincianfarini) [inline frontend/src/models/ActionInfo.ts:8]: This is not something that's included in the API. @jacknugent

Comment (jacknugent) [reply frontend/src/models/ActionInfo.ts]: Sorry didn't read your back-end code correctly, removed

Comment (kevincianfarini) [reply frontend/src/models/ActionInfo.ts]: All good, thanks.

File: backend/src/main/kotlin/com/climatechangemakers/act/feature/findlegislator/model/Legislator.kt (hunk 1 above)
Comment (jacknugent) [reply backend/src/main/kotlin/com/climatechangemakers/act/feature/findlegislator/model/Legislator.kt:10]: Adjusted the front-end type in [this commit](https://github.com/climatechangemakers/climate-changemakers-act/pull/69/commits/c845c732166f2779f351162864ca2325f02bd5dc)

**Review by jacknugent — APPROVED** [review]

**Pull request merged**